buf.gen.yaml

```diff
@@ -6,7 +6,7 @@ plugins:
     out: ./docs/
     opt: docs/scip.sprig,scip.md
     # NOTE: the sprig file is a template, it is not generated.
-  - remote: buf.build/library/plugins/go:v1.27.1-1
+  - plugin: buf.build/protocolbuffers/go:v1.30.0
```

Comment:
Newer versions of the `buf` CLI use the `plugin` key for both local and remote plugin references.